packages/flutter_tools/lib/src/web/web_device.dart

```diff
@@ -265,13 +296,29 @@ class WebDevices extends PollingDeviceDiscovery {
         operatingSystemUtils: operatingSystemUtils,
       ),
     );
+    if (platform.isWindows) {
+      _edgeDevice = MicrosoftEdgeDevice(
```

Comment:
recommendation: Since this is the new Edge, this will should trigger Chrome(blink) paths for browser related branches on Flutter Web Engine. Shall we add a note around it?

Reply:
I'm not really sure what you mean @nturgut ?

Reply:
Today a Flutter for Web engine developer can locally run edge tests using `felt` tool.
One scenario is, the developer sees a failing test when they run:
`felt test --browser=edge test\abc_test.dart` Let's say they try to reproduce the error by running flutter.  They run `flutter run -d edge`. But this time it is successful.

In order to not to confuse the developer in this way, we might add a comment (or a note to the README for flutter run) saying, `edge` option is running chromium based edge not the legacy edge.

So far I saw many issues filed for legacy edge, therefore I want to make sure those developers do not get confused.

Reply:
I'm confused how someone would end up in that scenario, either they have old edge installed or they have chromium edge?

Reply:
That's a good point indeed. There is a way of keeping the old edge for corporate customers but probably that would be a very small proportion.